tests/unit/controllers/notifications-test.js

```diff
@@ -0,0 +1,21 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'website-www/tests/helpers';
+import { NOTIFICATIONS } from 'website-www/constants/notifications';
+
+module('Unit | Controller | notifications', function (hooks) {
+  setupTest(hooks);
+
+  test('it exists', function (assert) {
+    const controller = this.owner.lookup('controller:notifications');
+    assert.ok(controller);
+  });
+
+  test('notifications property is set correctly', function (assert) {
+    const controller = this.owner.lookup('controller:notifications');
+    assert.deepEqual(
+      controller.notifications,
+      NOTIFICATIONS,
+      'notifications should be set to NOTIFICATIONS constant',
```

Comment:
We should ideally not be testing any internal states of a controller. While this adds some value, it also makes it harder to make updates to the controller class. We should focus on inputs and outputs, especially from a business or user point of view.